lib/vsc/install/shared_setup.py

```diff
@@ -1513,6 +1518,28 @@ def parse_target(self, target, urltemplate=None):
                     new_target['dependency_links'] += [''.join([git_scheme, url, '/hpcugent/', dep_name, '.git#egg=',
                                                                 dep_name_version])]
 
+        if 'VSC_RPM_PYTHON' in os.environ:
+            for key, snrs in vsc_filter_rpm.items():
+                def snr(txt):
+                    for pattern, replace in snrs:
+                        txt = re.sub(pattern, replace, txt)
```

Comment:
If an earlier replace is matched by a later pattern, this may not go right :p

Reply:
i know. wasn't sure if you want to stack the subs or go to the next as soon as a sub modifid the content

Reply:
I think it makes sense to keep going. The regular expressions are specified via a list, so you have control over the order in which they're processed.